### PR TITLE
avoid calling smcli to check fcp status when no fcp_list is configured

### PR DESCRIPTION
**Pull request by dongyanyang**
Signed-off-by: dyyang <dyyang@cn.ibm.com>

### CODE REVIEW

**Review by SharpRazor — APPROVED** [review]
lgtm.

File: zvmsdk/volumeop.py [hunk 1]
```diff
@@ -895,14 +895,20 @@ def _sync_db_with_fcp_list(self):
...
-        LOG.info("Enter: Sync FCP DB with FCP info queried from zVM.")
-
-        # Get a dict of all FCPs in ZVM
-        fcp_dict_in_zvm = self.get_fcp_dict_in_zvm()
+        LOG.info("Enter: Sync FCP DB with FCP info queried from z/VM.")
 
         # Get a dict of all FCPs in FCP DB
         fcp_dict_in_db = self.get_fcp_dict_in_db()
 
+        if fcp_dict_in_db == {}:
```
Comment (jackydalong) [inline zvmsdk/volumeop.py:903]: Not python style, please change to the following
```python
if not fcp_dict_in_db
```

File: zvmsdk/volumeop.py [hunk 2]
```diff
@@ -895,14 +895,20 @@ def _sync_db_with_fcp_list(self):
...
+            LOG.info("No FCPs exist in DB. ")
```
Comment (jackydalong) [inline zvmsdk/volumeop.py:904]: change to 
No FCPs exist in `FCP` DB

**Review by jackydalong — APPROVED** [review]
LGTM

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**